packages/encryption/src/lib/schemas/index.ts

```diff
@@ -0,0 +1,28 @@
+import { JSONSchemaType } from 'ajv';
+
+import { LIT_ERROR } from '@lit-protocol/constants';
+import { throwError } from '@lit-protocol/misc';
+import { ConditionType } from '@lit-protocol/types';
+
+export const SCHEMAS: { [K in ConditionType]: JSONSchemaType<any> } = {
+  cosmos: require('./LPACC_ATOM.json'),
+  evmBasic: require('./LPACC_EVM_BASIC.json'),
+  evmContract: require('./LPACC_EVM_CONTRACT.json'),
+  solRpc: require('./LPACC_SOL.json'),
+};
+
+export const getSchema = (
+  conditionType: ConditionType
+): JSONSchemaType<any> => {
```

Comment:
do we need a generic here if we are just saying `any` maybe we can restrict to the known schema types.

Reply:
The `any` is just because Ajv forces it. We care that it is a JSONSchemaType but it does not accept our derived types because it expects some more fields